### DISCUSSION TITLE
[hyper 0.14] Is 0.14.29 coming any time soon?

### DISCUSSION THREAD

**Discussion by yanakad** [hyperium/hyper#3634] [Q&A] [answered]
Hi all, there have been some h2 vulnerabilities lately that were patched.
We have a large codebase using Hyper0.14.X and have unfortunately so far been unsuccessful migrating to the new hyper1 APIs.

Could someone share if the 0.X versions will be maintained and for how long? It's been about 3months since 0.14.28 was cut and I'm starting to get worried that perhaps the 0.X codeline is nearing EOL?

**Comment by seanmonstar** [comment, marked as answer]
You can upgrade the h2 releases without a new hyper 0.14.x release. Just run `cargo update -p h2`.
